docs/modules/concepts/pages/storage/index.adoc

```diff
@@ -1,4 +1,34 @@
-= Mailboxes and Email Storage
+= Storage
```

Comment:
Mail storage in mailboxes makes sens for a mail delivery agent aka IMAP / POP3 / JMAP server.

However it don't make sense for mail servers that only relay emails.

This consideration IMO needs to appear here.

Reply:
Ok, good point.

Reply:
This change has already been made.